### DISCUSSION TITLE
How to set up exchange?

### DISCUSSION THREAD

**Discussion by Jackwaang** [jupyter/nbgrader#1878] [Q&A] [answered]
Hello,

I'm attempting to contribute to the repository as part of a class project. I'm having some issues testing functionality.

I'm trying to create an assignment and a student and have a student submit an assignment on my machine. 

I'm having trouble with setting up a file exchange, where I'm always met with an error that the directory does not exist. I have a picture of the situation below. I'm attempting to do this after following the developer installation and creating a sample course. Could anyone assist me?

![image](https://github.com/jupyter/nbgrader/assets/122063529/8ab43781-5a9e-4eb7-b22e-6eaa69454a96)

**Comment by perllaghu** [comment]
Does the folder exist?

> The nbgrader exchange directory writable to everyone. MUST be preexisting.

**Reply by Jackwaang** [reply]
I've adjusted the commands from the spec a bit so files aren't being created on a virtual machine, but yes I think the folder exists:

![image](https://github.com/jupyter/nbgrader/assets/122063529/dd10cba7-2b51-43ec-829c-b595ebb3f192)

**Reply by Jackwaang** [reply, marked as answer]
Actually, from the thread:

https://github.com/jupyter/nbgrader/issues/1338

it appears that you cannot use the exchange functionality on windows?
